File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/PtrackingUrl.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class PtrackingUrl{

	@SerializedName("baseUrl")
	private String baseUrl;

	public void setBaseUrl(String baseUrl){
		this.baseUrl = baseUrl;
	}

	public String getBaseUrl(){
		return baseUrl;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/PtrackingUrl.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class PtrackingUrl : Parcelable, Serializable {
    @SerializedName("baseUrl")
    var baseUrl: String? = null
    override fun toString(): String {
        return "PtrackingUrl{" +
                "baseUrl = '" + baseUrl + '\'' +
                "}"
    }
}